packages/react-native-reanimated/src/css/platform/native/types/animation.ts

```diff
@@ -45,3 +45,7 @@ export type NormalizedSingleCSSAnimationSettings = {
   fillMode: CSSAnimationFillMode;
   playState: CSSAnimationPlayState;
 };
+
+// HOST OBJECTS
+
+export type CSSKeyframesHostObject = Record<string, never>; // Placeholder type as we don't need any properties from the host object for now
```

Comment:
Signing utility types is a good practice.

```suggestion
/**
* Placeholder type as we don't need any properties from the host object for now
*/
export type CSSKeyframesHostObject = Record<string, never> & "CSSKeyframesHostObject"
```

Reply:
Similar remark as [here](https://github.com/software-mansion/react-native-reanimated/pull/7027#discussion_r1958377618). This is not exposed anywhere, so I don't think that it must be well-documented but ok, will change that as well.